parse/grammar/KuneiformParser.g4

```diff
@@ -147,6 +156,29 @@ constraint:
     (IDENTIFIER| PRIMARY KEY? | NOT NULL | DEFAULT | UNIQUE) (LPAREN literal RPAREN)?
 ;
 
+//// TODO: rename to constraint once table_delcaration is removed
+inline_constraint:
+    PRIMARY KEY
+    | UNIQUE
+    | NOT NULL
+    | DEFAULT literal
+    | fk_constraint
+    | CHECK (LPAREN sql_expr RPAREN)
+;
+
+fk_action:
+    ON (UPDATE|DELETE)
+    (SET NULL
+    | SET DEFAULT
+    | RESTRICT
+    | NO ACTION
+    | CASCADE)
+;
+
+fk_constraint:
+    REFERENCES table=identifier (LPAREN column=identifier RPAREN) (fk_action (fk_action)*)?
```

Comment:
Probably better to do:

```
REFERENCES table=identifier (LPAREN column=identifier RPAREN) (fk_action (fk_action)?)?
```

so that only a max of 2 can be specified

Reply:
because we only support `on DELETE` and `on UPDATE`? make sense

Reply:
I think that is all postgres supports too.